pkg/3scale/amp/operator/redis_options_provider_test.go

```diff
@@ -223,7 +219,7 @@ func TestGetRedisOptionsProvider(t *testing.T) {
 			func() *component.RedisOptions {
 				opts := defaultRedisOptions()
 				opts.SystemImage = systemRedisImageURL
-				opts.SystemRedisPodTemplateLabels["com.redhat.component-version"] = "systemCustomVersion"
+				//opts.SystemRedisPodTemplateLabels["com.redhat.component-version"] = "systemCustomVersion"
```

Comment:
This commented line and others can be removed

Reply:
fixed